src/test/kotlin/pt/up/fe/ni/website/backend/controller/GenerationControllerTest.kt

```diff
@@ -328,15 +397,22 @@ class GenerationControllerTest @Autowired constructor(
     inner class CreateGeneration {
         @Test
         fun `should create a new generation`() {
-            mockMvc.post("/generations/new") {
-                contentType = MediaType.APPLICATION_JSON
-                content = objectMapper.writeValueAsString(GenerationDto("22-23", emptyList()))
-            }
-                .andExpect {
-                    status { isOk() }
-                    content { contentType(MediaType.APPLICATION_JSON) }
-                    jsonPath("$.schoolYear") { value("22-23") }
-                }
+            mockMvc.perform(
+                post("/generations/new").contentType(MediaType.APPLICATION_JSON).content(
+                    objectMapper.writeValueAsString(GenerationDto("22-23", emptyList()))
+                )
+            )
+                .andExpectAll(
+                    status().isOk,
+                    content().contentType(MediaType.APPLICATION_JSON),
+                    jsonPath("$.schoolYear").value("22-23")
+                )
+                .andDocument(
+                    documentation,
+                    "Create new generations",
+                    "This operation retrieves creates a new generation.",
```

Comment:
```suggestion
                    "This operation creates a new generation.",
```
I think it was there by mistake